From 6a9d47ce27452c56487b6c6d1a078abb7e904f67 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 24 Jun 2014 18:55:49 +0200
Subject: [PATCH] print out, what kernel command line parameters are used

(cherry picked from commit fbc4620f059afec6169533259e24bc06479d18f9)

Conflicts:
	modules.d/98systemd/dracut-cmdline.sh
---
 modules.d/98systemd/dracut-cmdline.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules.d/98systemd/dracut-cmdline.sh b/modules.d/98systemd/dracut-cmdline.sh
index 36406d9..8a5ab2f 100755
--- a/modules.d/98systemd/dracut-cmdline.sh
+++ b/modules.d/98systemd/dracut-cmdline.sh
@@ -15,6 +15,9 @@ type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
 [ -f /etc/initrd-release ] && . /etc/initrd-release
 [ -n "$VERSION" ] && info "dracut-$VERSION"
 
+info "Using kernel command line parameters:"
+getcmdline | vinfo
+
 getargbool 0 rd.udev.log-priority=info -d rd.udev.info -d -n -y rdudevinfo && echo 'udev_log="info"' >> /etc/udev/udev.conf
 getargbool 0 rd.udev.log-priority=debug -d rd.udev.debug -d -n -y rdudevdebug && echo 'udev_log="debug"' >> /etc/udev/udev.conf
 
